src/daemon/rpm-ostreed.socket

```diff
@@ -0,0 +1,9 @@
+[Unit]
+ConditionKernelCommandLine=ostree
+
+[Socket]
+ListenStream=/run/rpm-ostree/client.sock
+SocketMode=0600
```

Comment:
Need to change this to be 0666 *or* make the connections to it only run if we're root.

Also xref https://github.com/coreos/rpm-ostree/issues/3850